lib/networkCall.js

```diff
@@ -0,0 +1,17 @@
+const fetch = require('node-fetch')
+
+const networkCall = async (url, method = 'get', body = null, headers = null) => {
```

Comment:
Constructing the whole URL everytime is a tedious task. Can we accept the query params in an argument tot teh function?